katello/katello-change-hostname

```diff
@@ -237,6 +238,12 @@ def hammer_cmd(cmd, exit_codes=[0], message=nil)
   run_cmd("hammer -u #{@options[:username].shellescape} -p #{@options[:password].shellescape} #{cmd}", exit_codes, message)
 end
 
+def get_default_proxy_id
+  output = hammer_cmd("--output json capsule info --name #{@old_hostname}")
+  proxy = JSON.parse(output)
+  proxy["Id"]
```

Comment:
if you return `proxy` instead of `proxy["Id"]` you can use the other details of the proxy (URL) in the calling method :)

Reply:
What parts of the proxy url do I need in the capsule update call? Its using the new hostname, and the `get_default_proxy_id` is returning the proxy with the old hostname

Reply:
At the moment, none.
I was still thinking that we can make the update call a bit more flexible, based on the old data. But this does not really belong here into the PR and is an own discussion :)